server/index.js

```diff
@@ -41,6 +41,37 @@ app.post('/gallery/update', (req, res) => {
   }
 });
 
+app.delete('/gallery/:id/photo/:photo', (req, res) => {
+  db.deletePhoto(req.params.id, req.params.photo, (err) => {
+    if (err) {
+      res.status(400).send();
+    } else {
+      res.status(200).send();
+    }
+  });
+});
+
+app.put('/gallery/:id/photo/:photo/url/:url', (req, res) => {
+  db.updateImgURL(req.params.id, req.params.photo, req.params.url, (err) => {
+    if (err) {
+      res.status(400).send();
+    } else {
+      res.status(200).send();
+    }
+  });
+});
+
+app.post('/gallery/:id/url/:url', (req, res) => {
+  console.log('hello')
+  db.postPhoto(req.params.id, req.params.url, (err) => {
+    if (err) {
+      res.status(400).send();
+    } else {
+      res.status(200).send();
```

Comment:
I would change this status code to a 201 instead of 200 since your are creating something. This is definitely a nit pick, but I believe it attains to the REST protocol.